src/base/utils/foreignapps.cpp

```diff
@@ -66,6 +66,7 @@ namespace
                 info = {exeName, versionStr.left(idx)};
             }
             catch (const std::runtime_error &err) {
+                Q_UNUSED(err);
```

Comment:
You can just remove variable name in the line above.

Reply:
`catch (const std::runtime_error &)` or `catch (std::runtime_error)` ?

Reply:
done